components/tag-list/tag-list-item-mixin.js

```diff
@@ -71,24 +108,71 @@ export const TagListItemMixin = superclass => class extends superclass {
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
-		const container = this.shadowRoot.querySelector('.tag-list-item-container');
+		const container = this.shadowRoot.querySelector('.tag-list-item-content');
 		this.addEventListener('focus', (e) => {
 			// ignore focus events coming from inside the tag content
-			if (e.composedPath()[0] === this) container.focus();
+			if (e.composedPath()[0] !== this) return;
+			const event = new FocusEvent('focus', { bubbles: true, cancelable: true });
+			container.dispatchEvent(event);
+		});
+		this.addEventListener('blur', () => {
+			const event = new FocusEvent('blur', { bubbles: true, cancelable: true });
+			container.dispatchEvent(event);
 		});
-		this.addEventListener('blur', () => container.blur());
+		this.addEventListener('keydown', this._handleKeydown);
+	}
+
+	deleteItem() {
```

Comment:
This was in multi select and is useful for the demo. If this seems like something that should be kept, I'll document it in the README as something consumers can use.

Reply:
So I guess the expectation is that the consuming component would have some array of data, they would remove the item from their array, and re-render?  It seems like consumers wouldn't really have a need to call this, if the tag-list is always a product of it's state.  If we wanted to exclude this method, we could set up a demo component in the demo.  🤷‍♂️